python/tvm/meta_schedule/tune_context.py

```diff
@@ -129,3 +129,4 @@ def __init__(
             rand_state,
             num_threads,
         )
+        _ffi_api.TuneContextNodeInitialize(self)
```

Comment:
To be consistent on the C++ side, shall we have a separate method like below instead of initializing everything by default?

```python
def initialize(self): ...
```